src/azure-cli-core/azure/cli/core/_output.py

```diff
@@ -131,11 +131,17 @@ def _capitalize_first_char(x):
     @staticmethod
     def _auto_table_item(item):
         new_entry = OrderedDict()
-        for k in item.keys():
-            if k in TableOutput.SKIP_KEYS:
-                continue
-            if item[k] and not isinstance(item[k], (list, dict, set)):
-                new_entry[TableOutput._capitalize_first_char(k)] = item[k]
+        try:
+            for k in item.keys():
+                if k in TableOutput.SKIP_KEYS:
+                    continue
+                if item[k] and not isinstance(item[k], (list, dict, set)):
+                    new_entry[TableOutput._capitalize_first_char(k)] = item[k]
+        except AttributeError:
+            # handles odd cases where a string/bool/etc. is returned
+            if isinstance(item, bool):
+                item = 'Success' if item else 'Fail'
```

Comment:
Should we use 'True' / 'False' instead of 'Success'/'Fail'?

I'm thinking that what if there was a command that returned whether some resource existed and it returned True. In our table output, we would them print 'Success' (or 'Fail') which may not make sense in that context?